cmd/lenses-cli/main.go

```diff
@@ -38,12 +38,12 @@ const (
 
 var (
 	app = &bite.Application{
-		Name:            "lenses-cli",
-		Description:     "Lenses-cli is the command line client for the Lenses REST API.",
-		Version:         lenses.Version,
-		PersistentFlags: setupConfigManager,
-		ShowSpinner:     false,
-		Setup:           setup,
+		Name:        "lenses-cli",
+		Description: "Lenses-cli is the command line client for the Lenses REST API.",
+		Version:     lenses.Version,
+		//PersistentFlags: setupConfigManager,
```

Comment:
you have commented it but I see you use it here https://github.com/Landoop/lenses-go/pull/6/files#diff-514e19b9501e58b1a5f1a32e1be48862R136

Reply:
I set this later in main.go as we don't need the default flags for secrets as we don't speak to Lenses so no need for kerberos etc.